src/MultiSelect/index.js

```diff
@@ -0,0 +1,276 @@
+      snap: true,
+      triggerOffset: 0,
+      containerOffset: 0,
+      placement: "bottom-start",
+      possiblePlacements: ["top-start", "bottom-start"],
+    });
+
+  const hasSelectedItem = selectedItems.length > 0;
+
+  const renderTokens = () =>
+    selectedItems.map((itemComponent, i) => {
+      const tokenLabel = itemToString(itemComponent);
+      const isLastToken = i === selectedItems.length - 1;
+      return (
+        <span
+          key={`${i}-${tokenLabel}`}
+          className={cc([
+            "padding--y--xs",
+            {
+              "padding--right--s": isLastToken,
+              "padding--right--xs": !isLastToken,
+            },
+          ])}
+        >
+          <FieldToken
+            label={tokenLabel}
+            onDismiss={() => removeSelectedItem(itemComponent)}
+            {...getSelectedItemProps({
+              selectedItem: itemComponent,
+              i,
+            })}
+          />
+        </span>
+      );
+    });
+
+  return (
+    <div className="nds-multiselect" data-testid={testId}>
+      <input type="hidden" name={name} id={name} value={fieldValue} />
+      <DropdownTrigger
+        isOpen={isOpen}
+        labelText={label}
+        startContent={renderTokens()}
+        errorText={errorText}
+        labelProps={{ ...getLabelProps() }}
+        {...getToggleButtonProps(
+          getDropdownProps({
+            ...triggerProps,
+          }),
+        )}
+        style={{
+          ...triggerProps.style,
+        }}
+      />
+      {renderLayer(
+        <div
+          className={cc([
+            "nds-multiselect-list",
+            "bgColor--white",
+            {
+              "rounded--bottom": layerSide === "bottom",
+              "rounded--top": layerSide === "top",
+              [`nds-select-list--active--${layerSide}`]: isOpen,
+            },
+          ])}
+          {...getMenuProps(layerProps)}
+          style={{
+            width: triggerBounds?.width,
+            transform: `translateY(${layerSide == "top" ? "3px" : "-3px"})`,
+            ...layerProps.style,
+          }}
+        >
+          {isOpen && (
+            <ul className="list--reset">
+              {items.map((item, index) => (
+                <li
+                  key={`item-${index}`}
+                  className={cc([
+                    "nds-multiselect-item",
+                    "alignChild--left--center padding--x--s padding--y--xs",
+                    {
+                      "nds-select-item--highlighted":
+                        highlightedIndex === index,
+                      "rounded--top": index === 0,
+                      "rounded--bottom": index === items.length - 1,
+                      "nds-select-item--hasGutter": hasSelectedItem,
+                    },
+                  ])}
+                  {...getItemProps({ item, index })}
+                  role="option"
+                  aria-selected={isSelected(selectedItems, item).toString()}
```

Comment:
When using a `useMultiSelect` in combination with a `useSelect`, we get the incorrect `aria-selected` value on items in the listbox. Here I'm just overriding what the prop getter returns, using my own helper to determine if the item is selected or not.

The `role` prop is only needed here to expose the linter to it (it's otherwise added by `getItemProps`)